src/components/BasicDialog/BasicDialog.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import DialogTitle from '@mui/material/DialogTitle';
+import Dialog from '@mui/material/Dialog';
+import {DialogContent} from '@mui/material';
+
+interface BasicDialogProps {
+  open: boolean;
+  title: string;
+  onClose: () => void;
+  children: React.ReactNode;
```

Comment:
Use destructured  react imports, it doesn`t make any sense to import whole react into each component.